ci/container-image-build-task.yml

```diff
@@ -0,0 +1,28 @@
+# task that builds docker images with an option to specify a target platform.
```

Comment:
Should we add copy rights to this file too?

Reply:
I'd say than we should them add to all pipeline files, but yes, I can add those.